### PR TITLE
fix: add custom input date format when importing csv file

### LINKED ISSUE #3712: Importing Cointracking.info CSV file
## Problem Definition

I am having trouble importing cointracking's CSV file into rotki.

Every entry gives me this error: "Backend - Error during cointracking CSV import deserialization. Error was Failed to deserialize a timestamp from a null entry in cointracking.info. Ignoring entry."

Maybe cointracking has changed up the CSV fields and the timestamp field has been moved to a different spot?

I will attach part of the CSV file that was outputted from Cointracking

Thanks

[CoinTracking_Trade_Table.csv](https://github.com/rotki/rotki/files/7532602/CoinTracking_Trade_Table.csv)

import error

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #3712

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Add custom input date format when importing csv file

### CODE REVIEW

**Comment by yabirgb** [comment]
Thank @lukicenturi . The dropdown makes much better the UX in this section now in my opinion.

About the change for the timestamp I write it here to have it written. The backend has a default timestamp that is the expected one for that file and is not exposed. That's why the field is marked as optional in the API docs. When the timestamp attribute in the request is set to null the backend chooses the expected format.

I believe is better to have it as an option there like "Use custom timestamp" and if is not selected send null in the request, otherwise send what the user introduced

**Comment by yabirgb** [comment]
Thanks @lukicenturi ! Now it works as expected. For my part the last comment is that this format is not always the default, for each location it might be different. I would just remove that note
image

As a said for me is now correct (except that small comment). Let's see if @kelsos has any comment for the frontend code.

Thank you very much @lukicenturi  :)

File: frontend/app/src/components/import/GroupedImport.vue [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        item-value="identifier"
+        item-text="name"
+        :hide-details="true"
+      >
+        <template #item="{ item }">
+          <v-row v-if="item">
+            <v-col class="d-flex align-center justify-space-between col-12">
+              <div
+                :class="`${$style.image} ${
+                  dark ? $style['image--inverted'] : ''
```
Comment (kelsos) [inline frontend/app/src/components/import/GroupedImport.vue:18]: wouldn't the [object syntax](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax) make more sense here?

File: frontend/app/src/components/import/GroupedImport.vue [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                  :contain="true"
+                />
+              </div>
+              <div>{{ item.name }}</div>
+            </v-col>
+          </v-row>
+        </template>
+        <template #selection="{ item }">
+          <div
+            :class="`${$style.image} ${dark ? $style['image--inverted'] : ''}`"
```
Comment (kelsos) [inline frontend/app/src/components/import/GroupedImport.vue:35]: The same as above here?

Comment (lukicenturi) [reply frontend/app/src/components/import/GroupedImport.vue:35]: Hmm actually yes, is there any way to fill same content to those two template?

Comment (lukicenturi) [reply frontend/app/src/components/import/GroupedImport.vue]: I guess I have to use v-for for this?

**Review by kelsos — APPROVED** [review]
lgtm, thank you

File: frontend/app/src/components/import/GroupedImport.vue (hunk 2 above)
Comment (lukicenturi) [reply frontend/app/src/components/import/GroupedImport.vue]: updated @kelsos

**Pull request merged**